frame/contracts/proc-macro/src/lib.rs

```diff
@@ -316,9 +323,27 @@ impl EnvDef {
 			.ok_or(err("Invalid environment definition, expected `mod` to be inlined."))?
 			.1;
 
+		let extract_fn = |i: &syn::Item| match i {
+			syn::Item::Fn(i_fn) => Some(i_fn.clone()),
+			_ => None,
+		};
+
+		let selector = |a: &syn::Attribute| a.path.is_ident("prefixed_alias");
+
+		let aliases = items
+			.iter()
+			.filter_map(extract_fn)
+			.filter(|i| i.attrs.iter().any(selector))
+			.map(|mut i| {
```

Comment:
You could do: `item.sig.ident = quote::format_ident!("seal_{}", item.sig.ident);`

Reply:
would be nice but this requires `std`